make/include/registry

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+IMAGE_ORG=${IMAGE_ORG:-'splatform'}
+IMAGE_PREFIX=${IMAGE_PREFIX:-'uaa'}
+
+registry_state() {
+    if test -z ${IMAGE_REGISTRY+nil}; then
+        echo unset
+    else
+        if test -z ${IMAGE_REGISTRY:-''}; then
+            echo empty
+        else
+            echo set
+        fi
+    fi
+}
+
+case $(registry_state) in
+    set|unset)
+        IMAGE_REGISTRY=${IMAGE_REGISTRY%/}/
+        ;;
+    empty)
+        IMAGE_REGISTRY=''
+        ;;
+esac
```

Comment:
Done with commit 5997614d09568ee399cc3b5690a39650e5b2ab29
(And indeed, SCF was already sorted)